common/event_modifiers/ME_Timurid_Princes_Modifiers.txt

```diff
@@ -32,7 +32,7 @@ ME_Baysonqor = {
 	global_autonomy = -0.1
 }
 
-ME_khorasan_agaist_beg = {
+ME_khorasan_against_beg = {
```

Comment:
This is fine for now, since you corrected all instances. But in general, fixing spelling mistakes in variable names would be tiresome and ultimately redundant. There are a lot of these in the code, and changing them can break things - it's easy to miss a single instance, and debugging that from that point would be awful.

Reply:
I wouldnt dare do it manually, I used VSC to find them all and replace all. I am not a madman

Reply:
*shivers*